index.js

```diff
@@ -0,0 +1,118 @@
+                  <input type='submit' name='reserve' value='Reserve'/>
+                </form>`
+                );
+
+                // https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/10-JavaScript/reference/axios/index.js credit to these notes
+
+                const readFormData = () => {
+                  const parsedFormData = {};
+
+                  const nameFromForm = $(`#reservation-form input[name='name']`).val();
+                  parsedFormData['name'] = nameFromForm ? nameFromForm : undefined;
+
+                  const emailFromForm = $(`#reservation-form input[name='email']`).val();
+                  parsedFormData['email'] = emailFromForm ? emailFromForm : undefined;
+
+                  return parsedFormData;
+                };
+
+                const clearForm = () => {
+                  $(`#reservation-form input[name='name']`).val('');
+                  $(`#reservation-form input[name='email']`).val('');
+                }
+
+                const createReservation = (event) => {
+                  event.preventDefault();
+
+                  const reservationData = readFormData();
+                  console.log(reservationData);
+
+                  reportStatus('Sending reservation data...');
+                  const URL = `https://trektravel.herokuapp.com/trips/${trip.id}/reservations`
+
+                  axios.post(URL, reservationData)
+                    .then((response) => {
+                      reportStatus(`Successfully added a reservation with ID ${response.data.id} for ${trip.name}!`);
+                      clearForm();
+                    })
+                    .catch((error) => {
+                      console.log(error.response);
+                      if (error.response.data && error.response.data.errors) {
+                        reportStatus(
+                          `Encountered an error: ${error.message}`,
+                          error.response.data.errors
+                        );
+                      } else {
+                        reportStatus(`Encountered an error: ${error.message}`);
+                      }
+                    });
+                };
+                $('#reservation-form').submit(createReservation);
+              })
+              .catch((error) => {
+                reportStatus(`Encountered an error while loading s: ${error.message}`);
+                console.log(error);
+              });
+            }
+
+        $('#trips').append(`<tr><td id='${trip.id}'>${trip.name}</td></tr>`);
+
+        $(`#${trip.id}`).one('click', loadOneTrip);
+
+        $(`#${trip.id}`).hover(() => {
+          $(`#${trip.id}`).css({'color': '#ff4949', 'font-weight': 'bold'});
+        }, () => {
+          $(`#${trip.id}`).css({'color': 'black', 'font-weight': 'normal'});
+        });
+      });
+    })
+    .catch((error) => {
+      reportStatus(`Encountered an error while loading reservations: ${error.message}`);
```

Comment:
You should really have a method to report errors specifically.  It makes sense to have a generic method to do it.